<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorSettingsPage</class>
 <widget class="QWidget" name="EditorSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QFontComboBox" name="fontCmb"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="fontSizeSpin">
        <property name="minimum">
         <number>6</number>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Indents</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Tabstop width</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="tabStopWidthSpin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="replaceTabsChk">
        <property name="text">
         <string>Replace tabs with spaces</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="unindentChk">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>If &quot;Replace tabs with spaces&quot; option is checked, 
the current option lets backspace key to remove the 
whole indent instead of removind single whitespace</string>
        </property>
        <property name="text">
         <string>Backspace removes indent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showIndentsChk">
        <property name="text">
         <string>Show indents</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="showLineLengthChk">
          <property name="text">
           <string>Show line length indicator</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="lineLengthSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> symbols</string>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="hlCurLineChk">
        <property name="text">
         <string>Highlight current line</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Default font color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fontColorBtn">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Default background color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bgColorBtn">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Current line highlighting color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="curLineColorBtn">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Matched brace background color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="braceColorBtn">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Indents color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="indentColorBtn">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Markers color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="markerColorBtn">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showLineLengthChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineLengthSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceTabsChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>unindentChk</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hlCurLineChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hlCurLineChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>curLineColorBtn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
